src/tools/clippy/clippy_lints/src/implicit_hasher.rs

```diff
@@ -1,3 +1,4 @@
+// todo remove after fix is synced
 #![allow(rustc::default_hash_types)]
```

Comment:
Not 100% sure, but removing this during sync might fail the rust-lang/rust CI afterwards. Shouldn't this be changed to `cfg_attr(bootstrap, ...)`?